app/controllers/public/sessions.js

```diff
@@ -0,0 +1,28 @@
+import Ember from 'ember';
+import moment from 'moment';
+import { orderBy, pick, groupBy, keyBy, mapValues, last } from 'lodash';
+
+const { Controller, computed } = Ember;
+
+export default Controller.extend({
+  sessions: [{ shortAbstract: 'An introduction to the event', id: '1', title: 'Welcome to FOSSASIA', startAt: moment(new Date()).format('h:mm a / DD-MM-YYYY'), track: { color: 'green', fontColor: 'green', id: 1, name: 'Track 1' }, microlocation: { id: 2, name: 'Room 1' }, speakers: [{ shortBiography: 'Works for ORG 1', id: 0, city: 'Delhi', name: 'Arnold Singh', speakingExperience: 'GSOC 2015', organisation: 'ORG 1', longBiography: '', photo: { 'iconImageUrl': '' } }] },
+  { shortAbstract: 'Welcome to open source', id: '2', title: 'Introduction to Open source', startAt: moment(new Date().setMinutes(10)).format('h:mm a / DD-MM-YYYY'), track: { color: 'red', fontColor: 'white', id: 2, name: 'Track 2' }, microlocation: { id: 3, name: 'Room 5' }, speakers: [{ shortBiography: '', id: 0, city: 'Delhi', name: 'Tux', speakingExperience: 'Speaker at the high conference', organisation: 'ORG 2', longBiography: '', photo: { 'iconImageUrl': '' } }] }
+  ],
+  sessionsByTracks: computed('sessions.[]', 'selectedTrackId', function() {
+    var allSessions = mapValues(groupBy(orderBy(this.get('sessions'), 'startAt'), 'track.id'), function(v) {
```

Comment:
This will give an output like 

```js
{
    'Track name': []
}
```

Where `[]` is an array of sessions. You can use `each-in` in the template to loop through this and get both track name and the sessions.